### PR TITLE
Change mfp scales

### PR DESCRIPTION
**Pull request by RemiDesgrange**
Also switch from windows-terminating style to unix (CRLF to LF)

To be cherry-picked on docker-master

### CODE REVIEW

File: mapfishapp/print/config.yaml [hunk 1]
```diff
@@ -1,1034 +1,1027 @@
...
+  - 25000
+  - 50000
+  - 100000
+  - 250000
+  - 500000
+  - 1000000
+  - 2500000
+  - 5000000
+  - 10000000
+
```
Comment (fvanderbiest) [inline mapfishapp/print/config.yaml:31]: Initial range goes up to 500 000 000
I think you can add more levels to match

**Comment by RemiDesgrange** [comment]
Relevant change

```diff
- scales:
-   - 267
-   - 533
-   - 1066
-   - 2133
-   - 4265
-   - 8531
-   - 17062
-   - 34124
-   - 68247
-   - 136495
-   - 272989
-   - 545979
-   - 1091958
-   - 2183915
-   - 4367830
-   - 8735660
-   - 17471321
-   - 34942642
-   - 69885283
-   - 139770566
-   - 279541132
-   - 559082264
+ scales:
+   - 250
+   - 500
+   - 1000
+   - 2500
+   - 5000
+   - 10000
+   - 25000
+   - 50000
+   - 100000
+   - 250000
+   - 500000
+   - 1000000
+   - 2500000
+   - 5000000
+   - 10000000
+   - 25000000
+   - 50000000
```

File: mapfishapp/print/config.yaml (hunk 1 above)
Comment (fvanderbiest) [reply mapfishapp/print/config.yaml:33]: Event with your latest change, there's an order of magnitude missing...

Comment (fvanderbiest) [reply mapfishapp/print/config.yaml:33]: Still incomplete @RemiDesgrange

Comment (fvanderbiest) [reply mapfishapp/print/config.yaml:33]: Up (for the release) ;-)
Thanks

**Comment by fvanderbiest** [comment]
LGTM, please merge & update `docker-master` accordingly

**Comment by RemiDesgrange** [comment]
ported in docker-master: 05fcc3c

**Pull request merged**